demos/notebooks/demo_dendritic.ipynb

```diff
@@ -330,16 +384,16 @@
     "grid_shape = (np.ceil(np.sqrt(Nc/2)).astype(int), np.ceil(np.sqrt(Nc*2)).astype(int))\n",
     "plt.figure(figsize=np.array(grid_shape[::-1])*1.5)\n",
     "plt.imshow(montage(A, rescale_intensity=True, grid_shape=grid_shape))\n",
-    "plt.title('Montage of found spatial components');\n",
+    "plt.title(f'Montage of initial spatial components alpha={alpha}');\n",
     "plt.axis('off');"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Run CNMF to sparsify the components\n",
-    "The components found by the initialization have captured a lot of structure but they are not sparse. Using the `refit` command will sparsify them."
+    "## Run CNMF to refine the components\n",
+    "The components found by the initialization have captured a lot of structure but they are have not had time to settle into an optimal solution. Using the `refit` command will start with the above initial solutions as seeds, and clean up the solution, typically ending up with more sparse solutions especially if you are running `graph_nmf`."
```

Comment:
"they are have not had time"?

Reply:
I'll clarify not enough iterations to converge to solution.